backend/globaleaks/handlers/admin/staticfiles.py

```diff
@@ -121,7 +121,6 @@ def post(self, filename):
         uploaded_file = self.get_file_upload()
         if uploaded_file is None:
             self.set_status(201)
-            self.finish()
             return
```

Comment:
see above comment; i'ts a flow spec. is not an error condition but simply a condition where the fille is still not completed;

flow.js specs wants 200, 201 or 202